packages/shared/src/hooks/input/useMarkdownInput.ts

```diff
@@ -270,6 +272,8 @@ export const useMarkdownInput = ({
     startUploading();
   };
 
+  const onCloseMention = useCallback(() => setQuery(undefined), [setQuery]);
```

Comment:
`setQuery` is a state setter and React guarantees reference equality so no need to add it to dependency array. ESlint plugin should not complain about it.